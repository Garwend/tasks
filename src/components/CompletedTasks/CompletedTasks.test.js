import renderer from 'react-test-renderer';
import CompletedTask from './CompletedTasks';

test('should render CompletedTask component', () => {
    const tasks = [{id:1, title:'test', priority: 3, seconds: 5}]
    const tree = renderer.create(<CompletedTask completedTasks={tasks} />).toJSON();
    expect(tree).toMatchSnapshot();
})