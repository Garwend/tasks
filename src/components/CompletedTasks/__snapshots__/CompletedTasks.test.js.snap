// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CompletedTask component 1`] = `
<article
  className="tasks-wrap"
>
  <header>
    Skończone zadania
  </header>
  <ul>
    <li
      className="task"
    >
      <p>
        <span>
          czas: 
        </span>
         
        5
         sekund 
      </p>
      <p>
        <span>
          priorytet: 
        </span>
         
        3
      </p>
      <button
        className="delete-btn"
        onClick={[Function]}
      >
        usuń
      </button>
    </li>
  </ul>
</article>
`;
